Makefile

```diff
@@ -25,6 +25,7 @@ integration-tests: release
 	echo "Running Integration Tests"
 	# Commented for now until we fix emscripten
 	# ./integration_tests/nginx/test.sh
+	./integration_tests/lua/test.sh
```

Comment:
I think it should be added, but commented until emscripten is fully working.
Otherwise all tests will be failing